### PR TITLE
[ingresscontroller] Add Easegress IngressController

### PR DESCRIPTION
**Pull request by benja-wu**
Hi, This PR is aiming to introduce `Easegress` as a Kubernetes Ingress Controller into this Kubernetes website. 
1. We had implemented this feature in our open-source API gateway [Easegress](https://github.com/megaease/easegress)
2. The IngressController configuration and demoing doc is [here](https://github.com/megaease/easegress/blob/main/doc/ingresscontroller.md)

Thanks.

### CODE REVIEW

**Comment by tengqm** [comment]
By convention, we accept PRs target a single language. This one is proposing changes to 5 different languages.
I'm not sure how soon you can collect LGTMs from 5 different sub-teams.
/lgtm
for English and Chinese.

**Comment by benja-wu** [comment]
> By convention, we accept PRs target a single language. This one is proposing changes to 5 different languages.
> I'm not sure how soon you can collect LGTMs from 5 different sub-teams.
> /lgtm
> for English and Chinese.

@tengqm  Thanks for the reminder. 
Sorry for haven't read the PR rule carefully enough. I will break it into individual PR next time. 
Since it is in processing now, I think maybe it's acceptable to wait for all other teams' approval in this single PR.
Thanks.

File: content/ko/docs/concepts/services-networking/ingress-controllers.md [hunk 1]
```diff
@@ -32,6 +32,7 @@ weight: 40
   Citrix 애플리케이션 딜리버리 컨트롤러에서 작동한다.
 * [Contour](https://projectcontour.io/)는 [Envoy](https://www.envoyproxy.io/) 기반 인그레스 컨트롤러다.
 * [EnRoute](https://getenroute.io/)는 인그레스 컨트롤러로 실행할 수 있는 [Envoy](https://www.envoyproxy.io) 기반 API 게이트웨이다.
+* [Easegress IngressController](https://github.com/megaease/easegress/blob/main/doc/ingresscontroller.md)는 [Easegress](https://megaease.com/easegress/) API 기반 게이트웨이이며, Ingress 컨트롤러로 실행할 수 있습니다.
```
Comment (seokho-son) [inline content/ko/docs/concepts/services-networking/ingress-controllers.md:35]: Hi @benja-wu 
According to a convention regarding Korean localization, my suggestion is as follows,

```suggestion
* [Easegress 인그레스 컨트롤러](https://github.com/megaease/easegress/blob/main/doc/ingresscontroller.md)는 [Easegress](https://megaease.com/easegress/) API 기반 게이트웨이이며, 인그레스 컨트롤러로서 실행될 수 있다.
```

I hope to note that Korean localization team has own development branch (currently, dev-1.21-ko.7), 
we accept only bug fix PR to main branch in general. But this PR includes some Korean translations. 

Anyway, I think you can 
- continue to proceed in this PR (it is not regular Korean localization contribution process), 
- open an individual PR to dev-1.21-ko.7, or 
- just remove changes in Korean file of this PR. (Korean localization team ourself will follow up English changes within 3 weeks)

**Review by sftim — COMMENTED** [review]
This will be easiest review if you just make English changes - that way, we can get the change merged and localization teams can pick up the update on their own timetable.

(@benja-wu you can propose those localization updates yourself as you want, but as separate PRs).

File: content/ko/docs/concepts/services-networking/ingress-controllers.md (hunk 1 above)
Comment (benja-wu) [reply content/ko/docs/concepts/services-networking/ingress-controllers.md]: Thank you for the modification. I will move it into an new individual PR for the Korean doc update.  :-)

**Comment by benja-wu** [comment]
@seokho-son  @sftim 
Thank you very much for the review and suggestions.
I had updated this PR to include the `ch` and `en` versions changes only(which had gained `lgtm` from tengqm).

**Comment by sftim** [comment]
/lgtm
(echoing @tengqm)
/approve

**Pull request merged**